### PR TITLE
Feature/improve downloads

### PR DESCRIPTION
**Pull request by simaob**
Following this thread: https://basecamp.com/1756858/projects/16557827/messages/88926209#comment_736046525

I'm going to check if we have the units on production as it should be showing up.

### CODE REVIEW

**Comment by simaob** [comment]
yes, we didn't run the units import. Will add to the reimport task, was on the seeds.

**Comment by tsubik** [comment]
Taking a look.

File: app/controllers/tpi/sectors_controller.rb [hunk 1]
```diff
@@ -53,6 +53,8 @@ def user_download_all
       companies_ids = Company.published.select(:id).where(sector_id: @sectors.pluck(:id))
       mq_assessments = MQ::Assessment
         .where(company_id: companies_ids)
+        .joins(:company)
+        .order('companies.name ASC, assessment_date DESC')
```
Comment (tsubik) [inline app/controllers/tpi/sectors_controller.rb]: I think that should be done everywhere.

Comment (tsubik) [reply app/controllers/tpi/sectors_controller.rb]: we could sort it in ruby code, directly in exporters I think.

Comment (tsubik) [reply app/controllers/tpi/sectors_controller.rb]: or maybe better, with sql

**Comment by tsubik** [comment]
I can take this over @simaob to change order everywhere and test more

**Comment by simaob** [comment]
Thanks, do it!

**Comment by simaob** [comment]
Is it worth doing further changes in a different PR, so that I can get this version into prod?

**Comment by tsubik** [comment]
one sec

**Comment by tsubik** [comment]
Done

**Comment by tsubik** [comment]
Wait

**Comment by tsubik** [comment]
Maybe CP benchamarks needs order too

**Comment by tsubik** [comment]
one more sec

**Comment by tsubik** [comment]
Done

**Pull request merged**